src/jsx_config.erl

```diff
@@ -342,7 +348,9 @@ config_to_list_test_() ->
                 stream,
                 uescape,
                 unescaped_jsonp,
+                tuples_to_lists,
                 strict
+
```

Comment:
Would it be OK to revert this empty line?

Reply:
yeah